agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of the Agent
-const Version = "1.86.3"
+const Version = "1.87.0"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "d6e4c7a1"
+const GitShortHash = "f398e88d"
```

Comment:
this should be `c4616021`: commit ID of the latest commit in the [dev branch](https://github.com/aws/amazon-ecs-agent/tree/dev).

your local branch may not be up-to-date with the [last PR](https://github.com/aws/amazon-ecs-agent/pull/4353) that was merged into dev

Reply:
Thanks for catching it